src/openrct2/localisation/FormatCodes.cpp

```diff
@@ -45,6 +45,7 @@ static const EnumMap<FormatToken> FormatTokenMap = {
     { "DURATION",             FormatToken::DurationShort,       },
     { "REALTIME",             FormatToken::DurationLong,        },
     { "LENGTH",               FormatToken::Length,              },
+    { "HEIGHT",               FormatToken::Height,              },
```

Comment:
Could you explain this addition? It is unusual that a plugin PR would need to add a formatting token.

Reply:
There was a conversation a while ago where we were talking about what units to return for height, since the game stores the height in height units. I'm pretty sure we decided to return these as height units and let scripters change them through the formatting function, since we return height units elsewhere (see [https://github.com/OpenRCT2/OpenRCT2/pull/20539#discussion_r1259926459](discussion))

Reply:
This PR has been ongoing for a REALLY long time for a relatively simple addition, is this + [this comment](https://github.com/OpenRCT2/OpenRCT2/pull/20539#discussion_r1545492165) the last steps needed before being merged?

I barely even play the game anymore 😂

Reply:
Sorry for my late reply, we are considerably understaffed and reviewing third-party PRs often falls through the cracks. I’ll try and take a look after the v0.4.11 release.